.github/actions/haos-builder-command/action.yml

```diff
@@ -0,0 +1,22 @@
+name: "Run command in HAOS build container"
+inputs:
+  image:
+    description: "HAOS builder image to use"
+    required: true
+  command:
+    description: "Command to run in the container"
+    required: true
+runs:
+  using: 'composite'
+  steps:
+  - name: "Run command in HAOS build container"
+    shell: bash
+    run: |
+      docker run --rm --privileged \
+        -e BUILDER_UID="$(id -u)" \
+        -e BUILDER_GID="$(id -g)" \
+        -v "${GITHUB_WORKSPACE}:/build" \
+        -v "/mnt/cache:/cache" \
+        -v "/mnt/output:/build/output" \
+        ${{ inputs.image }} \
+        ${{ inputs.command }}
```

Comment:
I wonder if we maybe could create a whole job which runs inside that container using the GitHub Action `container` syntax :thinking: 

https://docs.github.com/en/actions/writing-workflows/choosing-where-your-workflow-runs/running-jobs-in-a-container

Not sure if it works as all steps have then to run inside the container. I guess we could move some things inside the container (like the AWS upload, then we wouldn't need to install that every time), but some are probably easier handled outside (probably most other steps).

In any case, this is a step in the right direction. Moving to the native `container` support can be explored separately.